### ISSUE TITLE
 It is unclear whether the P2G units produce for domestic use or for export.

### ISSUE THREAD

**Issue by DorinevanderVlies** [quintel/etmodel#2257]
### Problem description

When I use the P2G flexibility options to reduce my excess electricity my aim is to supply the hydrogen demand, but not to overproduce hydrogen (which results in export).

For example the Urgenda scenario does not contain hydrogen technologies in the transport sector. So, implementing P2G for transport directly results in export of hydrogen.

Hydrogen is used non-energetically in the industry sector. Currently I use some educated guessing/trial-and-error to find the slider setting with which all hydrogen demand in the fertilizer industry sector is supplied through P2G without overproducing (exporting) hydrogen.

In short, it is unclear whether the P2G units produce for domestic use or for export. This can be seen, but requires quite a lot of ETM knowledge of the user.
It is also unclear how many P2G units would sufficient to fulfill the hydrogen demand in the transport and industry sector.
### Proposal
- Add a calculation/chart that indicates the number of P2G units that would fulfill the hydrogen demand in the transport an industry sector. _I do not yet have a proposal for the implementation of this._
- Adjust slide/slider text to contain information about the non-energetic use of hydrogen in industry. This is related to #2211.

**Comment by jorisberkhout** [comment]
Removing from milestone as this is more complicated to solve than I anticipated.

**Comment by jorisberkhout** [comment]
This is solved by @MichielDenHaan by changing the existing chart for hydrogen in transport and adding a similar graph for hydrogen in the fertilizer industry. See below.

<img width="607" alt="screen shot 2017-10-31 at 10 00 10" src="https://user-images.githubusercontent.com/4669028/32215549-8db3172e-be22-11e7-9e6f-6c6f82568835.png">
<img width="610" alt="screen shot 2017-10-31 at 10 00 45" src="https://user-images.githubusercontent.com/4669028/32215550-8dcac9d2-be22-11e7-9d3e-0c325326f9b2.png">

**Comment by DorinevanderVlies** [comment]
Nice!
@MichielDenHaan and @jorisberkhout I like this a lot!
